gthockey-iOS/gthockey-iOS/CustomUIComponents/PillButton.swift

```diff
@@ -57,4 +58,36 @@ class PillButton: UIButton {
         fatalError("init(coder:) has not been implemented")
     }
 
+    // MARK: Loading Animation
+
+    private var originalButtonText: String?
+
+    private lazy var activityIndicator: UIActivityIndicatorView = {
+        let activityIndicator = UIActivityIndicatorView()
+        activityIndicator.translatesAutoresizingMaskIntoConstraints = false
+        activityIndicator.color = .black
+        addSubview(activityIndicator)
```

Comment:
Let's move the `addSubview()` into the last line of `override init(frame: CGRect)` func. Make sure to call `updateConstraints()` just after (see next comment).

*This is similar to the flow in classes like `SignupView.swift` and `LoginView.swift`*